cmd/webhook/main.go

```diff
@@ -169,6 +171,13 @@ func newConversionController(ctx context.Context, cmw configmap.Watcher) *contro
 		// conversions to and from all types.
 		// "Zygotes" are the supported versions.
 		map[schema.GroupKind]conversion.GroupKindConversion{
+			v1alpha1.Kind("StepAction"): {
```

Comment:
We can add the conversion webhook when we need it?

Reply:
We need this for validation web hook. However, I removed the `Zygotes`.